container/view_test.go

```diff
@@ -46,7 +47,7 @@ func TestViewSaveDelete(t *testing.T) {
 		t.Fatal(err)
 	}
 	c := newContainer(t)
-	if err := c.CheckpointTo(db); err != nil {
+	if err := c.CheckpointTo(context.Background(), db); err != nil {
```

Comment:
I know we started adding traces in our integration tests; would we be doing that for unit tests as well in future?

Not sure if that makes sense for unit tests, so perhaps it's just a silly comment; but if it's not a silly question, we should probably also add a `ctx := context.TODO()` at the start that we can wire up in future.

https://github.com/moby/moby/blob/f3f20c3a862ff90c7e7b0ee07dabe2176ed0a170/integration/image/tag_test.go#L12-L13

Reply:
> I know we started adding traces in our integration tests; would we be doing that for unit tests as well in future?

No, I don't think we would do that in the future. First, because OTel would add a measurable overhead to an already slow CI. Second, because they wouldn't add much value IMO.